### DISCUSSION TITLE
GitHub Student developer pack sig  up issues.

### DISCUSSION THREAD

**Discussion by michael00921** [community/community#56963] [GitHub Education]
### Hi everyone,

Hey everyone. so I have had a GitHub student account for a very long time, but for some reason, I lost access to the account email, thus losing access to the services Student Developer Pack offered. Now, I tried to sign up again. I signed up but it said that I have already signed up. fair, so I sent an email to GitHub and asked to either terminate the account and remove the information for me to be able to sign up again or help me out. I didn't get it for like 2 months at this point. so I now requested an email (an official school email issued by the fvcking government) ad I got it (grigalashvili.misha@students.gov.ge), I also used a new email. I also requested new documentation from the school in a different format. now I went to the signup page and I declined, saying I had to fill out my GitHub account, I had to choose the correct documentation from the drop list, had to wait a few days (since it was a new account), and the most annoying one, THE SYSTEM CANT SEE THE NAME OF THE SCHOOL. the only thing in the English language is the school name in the top right corner with VERY big letters and I t can see it. I don't know, I have a project to do and I would use some help. no, I got away with some but it still says to finish my GitHub account, make the name visible, and also the thing should be dated (which it isn't but it states in which class I am in, so it IS dated. The last time it was like this without the actual date and it worked). The last time I did this, a REAL HUMAN received it and I got in, now the system won't let me pass. is there any way around it? I cannot contact GitHub and I need it. 

Thank you

-Misha (Misho, whatever)

**Comment by samus-aran** [comment]
Hi @MishoMilo 👋 

Let's see how we can work through your rejection reasons. Do you have a screenshot of the latest rejection reasons? This might help someone give you a nudge in the right direction.
